### ISSUE TITLE
library names for windows install

### ISSUE THREAD

**Issue by dfagnan** [pluskid/Mocha.jl#75]
Related to a google julia-users thread - do we need to correct some of the libraries for Windows installation? I get could not load module "libcuda" errors. Upon attempting to initialize the backend.

```
julia> init(backend)
ERROR: error compiling init: error compiling init: could not load module libcuda
: The specified module could not be found.
```
https://groups.google.com/forum/#!topic/julia-users/H19Mj217O7g

**Comment by pluskid** [comment]
@dfagnan I updated the library name according to the thread you mentioned. Are you able to test whether it works on Windows? Thank you very much!

**Comment by pluskid** [comment]
Should be resolved by #106
